Resources/Prototypes/Adventure/_CyberneticGenesis/LilaKebab/meals.yml

```
@@ -0,0 +1,31 @@
+- type: entity
+  name: шаурма ли ла кебаб
+  parent: FoodMealBase
+  id: FoodMealLilaKebab
+  description: Выглядит сочным и аппетитным, и большим, как раз под ваш рот
+  components:
+  - type: FlavorProfile
+    flavors:
+      - tomato
+      - meaty
+      - spicy
+  - type: Sprite
+    state: shawarma
+  - type: Tag
+    tags:
+    - Meat
+  - type: SolutionContainerManager
+    solutions:
+      food:
+        maxVol: 48
+        reagents:
+        - ReagentId: Nutriment
+          Quantity: 19
+        - ReagentId: Protein
+          Quantity: 13
+        - ReagentId: BbqSauce
+          Quantity: 3
+        - ReagentId: Ketchup
+          Quantity: 3
+        - ReagentId: Vitamin
+          Quantity: 4
```

```suggestion
        maxVol: 48
        reagents:
        - ReagentId: Nutriment
          Quantity: 19
        - ReagentId: Protein
          Quantity: 13
        - ReagentId: BbqSauce
          Quantity: 3
        - ReagentId: Ketchup
          Quantity: 3
        - ReagentId: Vitamin
          Quantity: 4
```
Здесь сумма всех реагентов выходит 42, не знаю почему лимит 48